src/index.js

```diff
@@ -1,10 +1,11 @@
 const express = require('express')
 require('./dataBase/mongoose')
-
-const app = express()
+const User = require('./models/user')
+const userRouter = require('./routes/user')
 
 const port = process.env.PORT
```

Comment:
The environment variables should be fetched and set in the config file and the value should be imported and used here.